src/Routers/routes/ShiftingRoutes.tsx

```diff
@@ -7,12 +7,14 @@ import BoardView from "@/components/Shifting/ShiftingBoard";
 import ListView from "@/components/Shifting/ShiftingList";
 
 import { AppRoutes } from "@/Routers/AppRouter";
-
-const getDefaultView = () =>
-  localStorage.getItem("defaultShiftView") === "list" ? "list" : "board";
+import { getDefaultView } from "@/Utils/viewStorageUtils";
 
 const ShiftingRoutes: AppRoutes = {
-  "/shifting": () => <Redirect to={`/shifting/${getDefaultView()}`} />,
+  "/shifting": () => (
+    <Redirect
+      to={`/shifting/${getDefaultView("defaultResourceView", "board")}`}
+    />
+  ),
   "/shifting/board": () => <BoardView />,
   "/shifting/list": () => <ListView />,
```

Comment:
What about Shifting and Resources tabs, they have two dedicated components for each list and board view

Should i call them in a single file and conditionally handle the views or let it be same

Reply:
https://github.com/ohcnetwork/care_fe/pull/9485#discussion_r1890664503